server/routes/employees.js

```diff
@@ -1,19 +1,35 @@
 var express = require("express");
```

Comment:
Wait what's going on here? This pr is only supposed to be for the login stuff, why is the saving employees stuff in here too? Did you branch off your saveemployees branch for this?